src/main/java/io/supertokens/totp/Totp.java

```diff
@@ -13,47 +33,143 @@
 import io.supertokens.totp.exceptions.LimitReachedException;
 
 public class Totp {
+    private static String generateSecret() throws Exception {
```

Comment:
where did you get the code for this function from?

Reply:
Wrote on my own based totp-otp docs.

Reply:
Please link to the source.

Reply:
See https://github.com/jchambers/java-otp

Added this in the code as a comment.